### ISSUE TITLE
Is nvdiffrast.rasterize + nvdiffrast.interpolate differentiable with the vertex offset? 

### ISSUE THREAD

**Issue by fyviezhao** [NVlabs/nvdiffrast#52]
I am working on a **2D** image alignment problem: given a source and a target image, I want to use linear blend skinning to deform the triangulated source image to match the target. Specifically, I use a neural network to predict the per-vertex offset of the source and obtain a skinned 2D mesh that is ready to be rasterized. Following the [triangle.py](https://github.com/NVlabs/nvdiffrast/blob/main/samples/torch/triangle.py), I use the `nvdiffrast.rasterize` along with the `nvdiffrast.interpolate` to rasterize the deformed 2D mesh (i.e, a collection of colored triangles), obtaining the deformed image. 

My question is, is the loss between the rasterized deformed image and the target image be differentiable with the vertex offset and further with the neural network parameter?

For better understanding, I summarize my problem as the pseudo-code below, any help is appreciated!
```python
src_im, dst_im = load_im(src_file, dst_file)
src_pos, src_tri, src_col = delaunay_triangulation(src_im)
optimizer = torch.optim.Adam(my_neural_network.parameters())
pos_offset = my_neural_network(src_im, dst_im, nn_param)
deformed_pos = src_pos + pos_offset
glctx = dr.RasterizeGLContext()
rast, _ = dr.rasterize(glctx, deformed_pos, src_tri, resolution=[256, 192])
deformed_im, _ = dr.interpolate(src_col, rast, src_tri)
loss = my_loss_func(deformed_im, dst_im)
optimizer.zero_grad()
loss.backword()
optmizer.step() # will I get the gradient correctly?
```

**Comment by fyviezhao** [comment]
Yes, I tried adding antialiasing at the end of the rendering pipeline and it worked well. Thanks again for your detailed explanation.
